src/buildMathML.js

```diff
@@ -341,6 +341,31 @@ groupTypes.op = function(group, options) {
     return node;
 };
 
+groupTypes.mod = function(group, options) {
+    var inner = [];
+
+    if (group.value.modType === "pod" || group.value.modType === "pmod") {
+        inner.push(new mathMLTree.MathNode(
+            "mo", [makeText("(", group.mode)]));
+    }
+    if (group.value.modType !== "pod") {
+        inner.push(new mathMLTree.MathNode(
+            "mo", [makeText("mod", group.mode)]));
```

Comment:
OK, then I’ll leave this alone for now. The revised commit has all the other requested changes.